pagina2.html

```diff
@@ -0,0 +1,169 @@
+                <p>Lorem ipsum dolor sit amet.</p>
+            </div>
+            <div class="cardio">
+                <img src="#">
+                <h4>Cardio Trainning</h4>
+                <p>Lorem ipsum dolor sit amet.</p>
+            </div>
+            <div class="imagen2">
+                <img src="#">
+            </div>
+            <div class="imagen-fondo">
+                <img src="#">
+            </div>
+        </div>
+        </section>
+        <!-- seccion 3/-->
+        <section>
+            <h2> Why choose us</h2>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sequi earum harum neque illo iure sunt.</p>
+            <div class="contenedor-slider">
+                <div class="item1">
+                    <h4>Tops</h4>
+                    <img src="#">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus, nostrum?</p>
+                </div>
+                <div class="item2">
+                    <h4>Bottoms</h4>
+                    <img src="#">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus, nostrum?</p>
+                </div>
+                <div class="item3">
+                    <h4>Hoodies and Jackets</h4>
+                    <img src="#">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus, nostrum?</p>
+                </div>
+                <div class="item4">
+                    <h4>Accesories</h4>
+                    <img src="#">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus, nostrum?</p>
+                </div>
+            </div>
+        </section>
+        <!-- footer-->
+        <footer>
+            <div class="contenedor-footer">
+                <!--footer lado izquierdo-->
+                <div class="footer-izquierdo">
+                    <div class="logo-footer">
+                        <img src="#">
+                    </div>
+                    <div>
+                        <img src="#">
+                        <h6>463 Round Table Drive, 
+                           Norwood, OH, Ohio</h6>
+                 </div>
+                    <div>
+                        <img src="#">
+                        <h6>+123456789</h6>
+                    </div>
+                    <div>
+                        <img src="#">
+                        <img src="#">
+                        <img src="#">
+                     </div>
+                </div>
+                <!-- footer lado derecho-->
+                <div class="footer-derecho">
+                    <h3>Category</h3>
+                    <ul>
+                        <li><a href="#"></a>First Link</li>
+                        <li><a href="#"></a>Second long Link</li>
+                        <li><a href="#"></a>Third Link</li>
+                        <li><a href="#"></a>Fourth Link</li>
+                    </ul>

```

Comment:
Estos elementos se repiten continuamente, forman un "componente", todo componente debe estar contenido por un elemento.

```suggestion
                    <div>
                        <h3>Category</h3>
                        <ul>
                            <li><a href="#"></a>First Link</li>
                            <li><a href="#"></a>Second long Link</li>
                            <li><a href="#"></a>Third Link</li>
                            <li><a href="#"></a>Fourth Link</li>
                        </ul>
                     </div>
```

Reply:
Revisar 😄